scss/_grid.scss

```diff
@@ -23,10 +23,17 @@ Mobile First - This applies from 0px to 767px
 .row .row [class^="grid"]:only-child            { padding:0; }
 .row .row [class^="grid"]:last-child            { padding-right:0; }
 
-/***** Grid gutters *****/
-.grid, .grid-1, .grid-2, .grid-3, .grid-4,
-.grid-5, .grid-6, .grid-7, .grid-8, .grid-9,
-.grid-10, .grid-11, .grid-12                    { box-sizing:border-box; width:100%; }
+/***** Sets grid widths *****/
+.grid                                           { box-sizing:border-box; width:100%; }
+
+@for $i from 1 through $grid-columns            {
+  .grid-#{$i}                                   { box-sizing:border-box; width:100%; }
+}
+
+/***** Sets mobile override grid width *****/
+@for $i from 1 through $grid-columns {
+  .grid-#{$i}-mobile                            { flex-grow:$i; max-width:  $i / $grid-columns * 100%; }
```

Comment:
@retober We don't need this as why would a user need to have 12 columns on a screen size that small? I do have a class that allows the user to have 50% width columns ```.grid-6-mobile```